packages/admin-stories/src/docs/form/Layout.stories.tsx

```diff
@@ -0,0 +1,312 @@
+import {
+    CancelButton,
+    Field,
+    FieldContainer,
+    FinalFormCheckbox,
+    FinalFormInput,
+    FinalFormRadio,
+    FinalFormSelect,
+    FormSection,
+    OkayButton,
+} from "@comet/admin";
+import { createCometTheme } from "@comet/admin-theme";
+import type {} from "@comet/admin/src/themeAugmentation";
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    FormControlLabel,
+    Grid,
+    MenuItem,
+    Paper,
+    Typography,
+    withStyles,
+} from "@material-ui/core";
+import { ThemeProvider } from "@material-ui/styles";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import { Form } from "react-final-form";
+import styled from "styled-components";
+
+const fooBarOptions = [
+    { value: "foo", label: "Foo" },
+    { value: "bar", label: "Bar" },
+];
+
+const flavourOptions = [
+    { value: "chocolate", label: "Chocolate" },
+    { value: "strawberry", label: "Strawberry" },
+    { value: "vanilla", label: "Vanilla" },
+];
+
+storiesOf("stories/form/Layout", module)
+    .add("Fields in sidebar", () => {
+        const Root = styled.div`
```

Comment:
Creating styled components inside the render function of a story may cause some performance issues (basically, this is the same as a React function component). I'd move the styles outside of the story.

Reply:
but then you won't see it in the source when switching the story into source mode

Reply:
Good point! Then we'll leave it like this. Premature optimization... 😉